### DISCUSSION TITLE
Display the file path in the UI

### DISCUSSION THREAD

**Discussion by denilsonsa** [jellyfin/jellyfin#9930] [General] [answered]
When playing a certain media (video, music, books, playlists, …), it is sometimes useful to know the actual file path of the original file.

Can we please **display the original file path in the UI**? Either in the main player UI, or in the "Media Info" dialog.

---

For instance, I saw a duplicate album being listed in my Jellyfin music library. Okay, but where is this duplicate coming from?

I tried the ⋮ (overflow menu) on the album, but there is nothing that helps me in there.

![Music album overflow menu](https://github.com/jellyfin/jellyfin/assets/121676/625f5200-31e3-429a-a508-6fc383acdff6)

Then I tried the "Media Info" dialog on any individual track, but it's still not showing the path.

![Music track overflow menu](https://github.com/jellyfin/jellyfin/assets/121676/88daae54-cbcc-4c56-b947-f436f7ffbde0)

![Media Info dialog](https://github.com/jellyfin/jellyfin/assets/121676/bbbce3fb-2e1f-4542-84b1-577a97894f10)

I mainly use Jellyfin for listening to my music collection, I haven't tried the other kinds of media.

**Comment by crobibero** [comment, marked as answer]
The path is available to admin users. 

https://github.com/jellyfin/jellyfin-web/blob/97d1f71057c0672ce377eba31d7042e4b5bcd1ed/src/components/itemMediaInfo/itemMediaInfo.js#L67-L69
